rfcs/0000-rfc-template.md

```diff
@@ -23,6 +24,13 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 
 ## Fields
 
+The `rule` fields being proposed are as follows:
+
+Field | Type | Description /Usage
+-- | -- | -- | -- | -- 
```

Comment:
the number of ticks doesn't look correct

Reply:
Yes, the markdown table isn't rendering correctly right now.

Reply:
ACK